content/blog/2020/2020-04-20-wdl-head/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Win-Draw-Loss evaluation"
+slug: "wdl-head"
+published: 2020-04-20
+author: "crem"
+# image: "name"
+# cover: "cover.png" # Default
+loadChessbaseScripts: true
+---
+
+Traditionally, computer chess engines evaluated position in [pawns](https://www.chessprogramming.org/Centipawns).
+This is a convenient choice, as material is a foundation of position evaluation, which is true also among humans.
+Situations where importance of positional aspect strongly outweigh material are often thought as beautiful exceptions rather than a rule.
```

Comment:
**the** importance of positional aspect**s** strongly outweigh**ts**
rather than ~~a~~**the** rule